src/OmniSharp.Dnx/DnxPaths.cs

```diff
@@ -138,31 +132,24 @@ private IEnumerable<string> GetRuntimeLocations()
             {
                 // Newer path
                 yield return Path.Combine(home, ".dnx");
+            }
 
-                // New path
-                yield return Path.Combine(home, ".k");
-
-                // Old path
-                yield return Path.Combine(home, ".kre");
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("ProgramFiles")))
```

Comment:
I'm not at home at the moment, but I swear when I looked my rc1 install was in `C:\Program Files\Microsoft DNX`

I can add those variants as well though.